README.md

```diff
@@ -249,6 +254,13 @@ query ($id: String!) {
 
 <!-- TODO add screenshot -->
 
+### Schema Customization API
+
+To prevent errors from occuring when changes are made to the Contentful content model, this starter includes GraphQL type definitions in its [`gatsby-node.js`](gatsby-node.js) file.
+If you decide to make changes to your content model, be sure to update the type definitions in this file, otherwise the starter might not be able to query new or renamed fields.
+
+To read more about customizing, see the theme's [README.md][theme readme].
```

Comment:
I think this will be the source of a lot of bugs and needs more prominence some how. WDYT?

could we have some sort of call out, i think syntax is something like this:

```suggestion
> To prevent errors from occuring when changes are made to the Contentful content model, this starter includes GraphQL type definitions in its [`gatsby-node.js`](gatsby-node.js) file.
> If you decide to make changes to your content model, be sure to update the type definitions in this file, otherwise the starter might not be able to query new or renamed fields.
>
> To read more about customizing, see the theme's [README.md][theme readme].
```

Reply:
I'm unsure how big of an issue this would be, but it's the type of thing that might fail silently. To highlight this more prominently, we shouldn't use `blockquote` for things that aren't quotes, but one way to flag things in a markdown file is to add emoji like ⚠️ etc